components/centraldashboard/public/components/namespace-selector.js

```diff
@@ -113,6 +155,8 @@ export class NamespaceSelector extends PolymerElement {
      * @param {string} selected
      */
     _onSelected(selected) {
```

Comment:
I think this observer fires before before namespaces is assigned a value as I was seeing console errors when I deployed it. Probably easiest just to add a guard for namespaces being populated.

https://polymer-library.polymer-project.org/3.0/docs/devguide/observers#observers-and-element-initialization

Reply:
Fixed!